### ISSUE TITLE
В настройках скрипта было бы неплохо добавить опции

### ISSUE THREAD

**Issue by avesgit** [X2Online/AdsAutomation#1]
Не снимать галку "Не показывать проверенные объявления"
Логин, имя пользователя и пароль брать из конфига чтобы не мержить их

**Comment by X2Online** [comment]
Согласен, выделить конфиг в отдельный файл и расширить его – отличная идея.
